src/Command/UpdateCommand.php

```diff
@@ -94,7 +91,11 @@ protected function execute(InputInterface $input, OutputInterface $output)
             }
 
             //get versions as [version => url]
-            $versions = $info['versions'] ?: [];
```

Comment:
Open a new one for that as "cleanup" and it will be merged swiftly. Please move the rollingCurl change to it as well.